docs/UserGuide.md

```diff
@@ -134,18 +135,33 @@ Curious about the various commands? Refer to the [Features](#features) below to
 
 ## Before Using PennyWise
 
+### What is an Entry
+
+Think of PennyWise as a notebook that has infinite number of pages. You can write as much data as you want on your 
```

Comment:
```suggestion
Think of PennyWise as a notebook that has infinite number of pages. You can write as much data as you want to track your 
```

I feel that notebook may not be the appropriate analogy tho, are u referring to something like their financial spreadsheet or budgeting diary? I'm not sure how best to analogise it. What about personal budget assistant or something?

Reply:
Yea actually, that was what I meant but I was not sure how to analogise it as well HAHA, so I went ahead with notebook

Reply:
let me change it and you can review again.

Reply:
Sure, what do u think too? @sharmaine1028